### ISSUE TITLE
Upgrading pulsar client to 2.10.0 causing NoSuchMethodFoundError

### ISSUE THREAD

**Issue by asharma-97** [flipkart-incubator/pulsar-weighted-consumer#2]
We need to use 2.10.0 version of pulsar client. While upgrading pulsar client version, it is causing NoSuchMethodFoundError in pulsar weighted consumer with UnAckedTopicMessageTracker.

Here is error stacktrace:
```
java.lang.NoSuchMethodError: org.apache.pulsar.client.impl.UnAckedTopicMessageTracker.<init>(Lorg/apache/pulsar/client/impl/PulsarClientImpl;Lorg/apache/pulsar/client/impl/ConsumerBase;JJ)V
	at org.apache.pulsar.client.impl.WeightedMultiTopicsConsumerImpl.<init>(WeightedMultiTopicsConsumerImpl.java:142)
	at org.apache.pulsar.client.impl.WeightedMultiTopicsConsumerImpl.<init>(WeightedMultiTopicsConsumerImpl.java:112)
	at org.apache.pulsar.client.impl.WeightedMultiTopicsConsumerImpl.<init>(WeightedMultiTopicsConsumerImpl.java:96)
	at org.apache.pulsar.client.impl.WeightedMultiTopicsConsumerImpl.subscribeAsync(WeightedMultiTopicsConsumerImpl.java:313)
	at org.apache.pulsar.client.impl.WeightedConsumerBuilder.subscribeAsync(WeightedConsumerBuilder.java:149)
	at org.apache.pulsar.client.impl.ConsumerBuilderImpl.subscribe(ConsumerBuilderImpl.java:99)
	at fk.retail.srm.commons.pulsarClient.internal.PulsarPriorityConsumer.createConsumer(PulsarPriorityConsumer.java:40)
	at flipkart.sfm.torpedo.managed.PulsarConsumerManager.initializeConsumer(PulsarConsumerManager.java:56)
	at flipkart.sfm.torpedo.managed.PulsarConsumerManager.start(PulsarConsumerManager.java:44)
```

**Comment by grssam** [comment]
@asharma-97 Can you verify if #3 works for your use-case?

**Comment by grssam** [comment]
Fixed by #3
